modules/apps/data-engine/data-engine-js-components-web/test/js/__mock__/MetalFieldMock.soy.js

```diff
@@ -0,0 +1,106 @@
+/* jshint ignore:start */
+import Component from 'metal-component';
```

Comment:
This looks weird. We do have these types of artifacts in other places in DXP, but I wouldn't expect a new dependency version to force us into creating new artifacts.

Reply:
Yeah, I'm flabbergasted myself, I wanted to try to see if CI will pass with the changes, and the changes do look weird but locally `yarn test` passes...

Reply:
I might be onto something...